geos.go

```diff
@@ -57,6 +57,30 @@ var (
 	errIndexOutOfRange     = Error("index out of range")
 )
 
+type PrecisionRule int
+
+// PrecisionRules.
+const (
+	// The output is always valid. Collapsed geometry elements
+	// (including both polygons and lines) are removed.
+	PrecisionRuleValidOutput PrecisionRule = 0 // C.GEOS_PREC_VALID_OUTPUT
+	// Precision reduction is performed pointwise. Output geometry may
+	// be invalid due to collapse or self-intersection.
```

Comment:
There's no need to copy the comments from GEOS here. The users can refer to GEOS's documentation if needed. Note also that these comments do not follow [godoc conventions](https://go.dev/blog/godoc).